sandbox/tests/test scenes/sss sets/sss_sets_comparison.appleseed

```diff
@@ -0,0 +1,243 @@
+                <assign_material slot="None" side="front" material="marble_sss_material" />
+            </object_instance>
+            <object_instance name="light_left" object="sss_sets_test.Plane">
+                <parameter name="medium_priority" value="0" />
+                <parameter name="ray_bias_distance" value="0.0" />
+                <parameter name="ray_bias_method" value="none" />
+                <parameter name="use_individual_sss_set" value="true" />
+                <assign_material slot="None" side="back" material="light_material" />
+            </object_instance>
+            <object_instance name="light_right" object="sss_sets_test.Plane.001">
+                <parameter name="medium_priority" value="0" />
+                <parameter name="ray_bias_distance" value="0.0" />
+                <parameter name="ray_bias_method" value="none" />
+                <parameter name="use_individual_sss_set" value="true" />
+                <assign_material slot="None" side="back" material="light_material" />
+            </object_instance>
+            <object_instance name="sphere_left_front" object="sss_sets_test.Sphere.002_Sphere.001">
+                <parameter name="medium_priority" value="0" />
+                <parameter name="ray_bias_distance" value="0.0" />
+                <parameter name="ray_bias_method" value="none" />
+                <parameter name="use_individual_sss_set" value="true" />
+                <assign_material slot="None" side="front" material="marble_sss_material" />
+            </object_instance>
+            <object_instance name="sphere_left_side" object="sss_sets_test.Sphere.003">
+                <parameter name="medium_priority" value="0" />
+                <parameter name="ray_bias_distance" value="0.0" />
+                <parameter name="ray_bias_method" value="none" />
+                <parameter name="use_individual_sss_set" value="true" />
+                <assign_material slot="None" side="front" material="marble_sss_material" />
+            </object_instance>
+            <object_instance name="sphere_right_front" object="sss_sets_test.Sphere.001_Sphere.000">
+                <parameter name="medium_priority" value="0" />
+                <parameter name="ray_bias_distance" value="0.0" />
+                <parameter name="ray_bias_method" value="none" />
+                <parameter name="sss_set_id" value="sss_set" />
+                <assign_material slot="None" side="front" material="marble_sss_material" />
+            </object_instance>
+            <object_instance name="sphere_right_side" object="sss_sets_test.Sphere.004">
+                <parameter name="medium_priority" value="0" />
+                <parameter name="ray_bias_distance" value="0.0" />
+                <parameter name="ray_bias_method" value="none" />
+                <parameter name="sss_set_id" value="sss_set" />
+                <parameter name="use_individual_sss_set" value="false" />
```

Comment:
As far as I understand, this parameter no longer exist and needs to be removed from all test scenes.